platform/pulpcore/app/serializers/fields.py

```diff
@@ -23,7 +23,7 @@ class HrefWritableRepositoryRelatedField(serializers.HyperlinkedRelatedField):
     read_only should passed as a kwarg to this field.
     """
     view_name = 'repositories-detail'
-    lookup_field = 'name'
+    lookup_field = 'pk'
```

Comment:
Do you need to specify `pk` as a `lookup_field` explicitly? I think it's a default.

Reply:
I believe you are correct. I'll test without it and see if it still works.

Reply:
removing the lookup_field seems to work. however, i am not sure what other places in this PR could use a similar change.